vars/README.md

```diff
@@ -512,6 +512,20 @@ pipeline {
 }
 ```
 
+## findOldestStackVersion
+Find the oldest stack version given the condition to compare with.
+
+If the version doesn't exist yet, it will try to use the closer snapshot, for example
+if 7.14.1 doesn't exist, it will try to use 7.14.1-SNAPSHOT or 7.x-SNAPSHOT,
+this will allow to develop integrations with unreleased features.
+
+
+```
+findOldestStackVersion(versionCondition: "^7.14.0")
```

Comment:
I'm finding the name of this step very hard to understand. What it seems like this is doing is trying to find the _newest_ image relative to the version that's passed in. However, `findOldestStackVersion` makes it seem like it's looking for the old _least new_ image. Am I not understanding the purpose of this step correctly?

Reply:
The method is intended to find the oldest supported version for a given condition. Maybe `findOldestSupportedVersion` is better?

Reply:
I agree, I'm not also onboarded with the naming, maybe `findCloserStackVersion` could be the correct one.

@jsoriano implemented a specific function in https://github.com/elastic/integrations/pull/1541/files#r694781463 , I C&C the content and added a few UTs. Let's wait for Jaime's feedback and see if it can be changed with something more specific.

Reply:
> Maybe findOldestSupportedVersion is better?

This makes a lot of sense. Big 👍 from me for this change.

Reply:
See https://github.com/elastic/apm-pipeline-library/pull/1248/commits/7f20c354e38a7e72eb72e143b4dc9a5e2e93033e